packages/now-cli/src/util/dev/server.ts

```diff
@@ -1706,7 +1710,20 @@ export default class DevServer {
 
     this.output.debug(`Spawning dev command: ${command}`);
 
-    const p = spawnCommand(command, { stdio: 'inherit', cwd, env });
+    const p = spawnCommand(command, { stdio: 'pipe', cwd, env });
+
+    if (!p.stdin || !p.stdout || !p.stderr) {
+      throw new Error('Expected child process to have standard output');
+    }
+
+    process.stdin.pipe(p.stdin);
+    p.stderr.pipe(process.stderr);
+    p.stdout.setEncoding('utf8');
+    const devPort = new URL(this.address).port;
+
+    p.stdout.on('data', (data: string) => {
+      process.stdout.write(data.replace(port.toString(), devPort));
```

Comment:
Shouldn't the `port` be a global regexp to replace multiple instances?

Reply:
I changed it to a regex, thanks!